ci-operator/config/openshift/installer/openshift-installer-release-4.12.yaml

```diff
@@ -548,7 +548,7 @@ tests:
       DEVSCRIPTS_CONFIG: |
         AGENT_E2E_TEST_SCENARIO=COMPACT_IPV4
         AGENT_DEPLOY_MCE=true
-    workflow: agent-e2e-generic
+    workflow: agent-e2e-compact-ipv4
```

Comment:
This seems a typo, as the `DEVSCRIPTS_CONFIG` will override the one provided byu `agente2e-compact-ipv4` workflow

Reply:
Yeah, they both do the same thing today, since the compact workflow doesn't contain any config aside from DEVSCRIPTS_CONFIG.
It's arguable which one is more explicit.

Reply:
For the `generic` workflow is expected that the configuration will be (optionally) provided in the place where it's used, while for the specific workflows it's not expected to provided any additional config, so I'd keep the `generic` one